texastribune/donations
# A smattering of small UMP improvements

**Despite this PR having 50+ changed files, I'm really not changing much. That number is inflated just because I'm reordering a bunch of imports.**

#### What's this PR do?
+ Cleans up GTM event logging
+ Creates reusable components for "contact us" text
+ Adds a global mixin for date utilities (`dates.thisYear` and `dates.lastYear`, for now)
+ Makes it so all slots have descriptive names
+ Puts all `import` statements in a consistent order
+ Makes message components more robust/reusable

#### Why are we doing this? How does it help us?
This is the final bit of UMP maintenance I'll be tackling this quarter. I've done this all to ensure the app is ready to take on a bunch of new features later this year.

#### How should this be manually tested?
`make`, `make restart`, then just click around `/account/` and see if anything looks amiss.

#### How should this change be communicated to end users?
No user-facing changes.

#### Are there any smells or added technical debt to note?
Less.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/2417405467

#### Have you done the following, if applicable:
